prow/github/client.go

```diff
@@ -229,6 +229,26 @@ func (c *Client) DeleteComment(org, repo string, ID int) error {
 	return nil
 }
 
+func (c *Client) EditComment(org, repo string, ID int, comment string) error {
+	c.log("EditComment", org, repo, ID, comment)
+	if c.dry {
+		return nil
+	}
+
+	ic := IssueComment{
+		Body: comment,
+	}
+	resp, err := c.request(http.MethodPatch, fmt.Sprintf("%s/repos/%s/%s/issues/comments/%d", c.base, org, repo, ID), ic)
```

Comment:
you should really refactor this boilerplate into a single helper method, that probably returns the body if successful or else an error